src/hip_hcc.cpp

```diff
@@ -791,6 +791,7 @@ hipError_t ihipDevice_t::initProperties(hipDeviceProp_t* prop) {
     err = hsa_agent_get_info(_hsaAgent, HSA_AGENT_INFO_NAME, &archName);
 
     prop->gcnArch = atoi(archName + 3);
+    strcpy(prop->gcnArchName, archName + 3);
```

Comment:
Should the gcnArchName include the full name gfx900, or gfx1010, rather than just 900 or 1010? It might make sense for users to expect the correct name with gfx* prefix, as in AMDGPU guide. https://llvm.org/docs/AMDGPUUsage.html#processors

Reply:
Addressed the comments. gcnArchName is changed to include the full name with gfx prefix.